js/product/product.js

```diff
@@ -0,0 +1,21 @@
+angular.module("modelProduct", [])
+    .factory("localModelProduct", function(){
+
+    function Product(nazwa, cena_netto, cena_brutto){
+        this.nazwa = nazwa;
+        this.cena_netto = cena_netto;
+        this.cena_brutto = cena_brutto;
```

Comment:
bez sensu jest zapisywac cene brutto, skoro zawsze ja mozna wyliczyc. Nie zapisuj jej nigdzie jako osobne, i wywal to pole z kontruktora i z zawartosci klasy. Zawsze gdzie chcesz uzyc brutto, uzywaj metody getGrossPrice().
Jesli chcesz wyswietlic to co zwraca metoda w template, to robisz po prostu
{{ product.getGrossPrice() }} 

Tak samo nie pozwalaj uzytkownikowi edytowac ceny brutto - niech edytuje cene netto, a cena brutto sie przeliczy.